tests/TestCadQuery.py

```diff
@@ -1800,3 +1800,17 @@ def testClose(self):
 
         # The obj1 shape shall have the same volume as the obj2 shape.
         self.assertAlmostEqual(obj1.val().Volume(), obj2.val().Volume())
+        
+    def testText(self):
+      
+        box = Workplane("XY" ).box(4, 4, 0.5)
+        
+        obj1 = box.faces('>Z').workplane().text('CQ 2.0',0.5,-.05,cut=True)
+        
+        #combined object should have smaller volume
+        self.assertGreater(box.val().Volume(),obj1.val().Volume())
+        
+        obj2 = box.faces('>Z').workplane().text('CQ 2.0',0.5,.05,cut=False,combine=True)
```

Comment:
It's not really necessary, but you could check to make sure that you have the right number of faces as well. The Arial font should always produce a predictable number of extra faces when extruded.

Reply:
Good idea. Done